lib/sidekiq/form526_backup_submission_process/submit.rb

```diff
@@ -69,9 +69,12 @@ class Submit
         raise e
       end
 
+      # rubocop:disable Metrics/MethodLength
```

Comment:
Nit: I might consider breaking this up if you can, both for the linter and because it's just a little hard to read